### ISSUE TITLE
what is difference between weights="densenet121-res224-mimic_ch" and weights="densenet121-res224-mimic_nb"

### ISSUE THREAD

**Issue by catfish132** [mlmed/torchxrayvision#106]
I have used model with weights="densenet121-res224-mimic_ch"  to extract MIMIC-CXR's picture's feature ,and I found there 7 dims of the feature always equal to 0.5 ,it was strange.
I am new for medical image processing, and I found there two types of pre-trained model for MIMIC-CXR , What is the difference between them.
By the way , I have also used model with weight="densenet121-res224-all" , and I found this don't always present 0.5 in some dimensions like "densenet121-res224-mimic_ch" dose. Is the model with weight="densenet121-res224-all" suitable for all of the medical Image Dataset?

**Comment by ieee8023** [comment]
The 'ch' and 'nb' indicates which set of labels was used. That research team used two different systems to extract the label from the clinical notes. Each system may have different artifacts so they studied it in the paper for the MIMIC dataset. You can find descriptions of the datasets in the doc string of the datasets in datasets.py.

I would use the all weights in general. The other models can be used to study biases from the different training data.

**Comment by catfish132** [comment]
OK, I got it ,thanks a lot.
